.github/workflows/visualize.yml

```diff
@@ -0,0 +1,177 @@
+name: visualize
+on:
+ push:
+  branches:
+    - main
+ workflow_dispatch:
+
+jobs:
+  run-must:
+    runs-on: ubuntu-latest
+    env:
+      MPI_CORRECTNESS_BM_DIR: ${{ github.workspace }}
+      MPI_CORRECTNESS_BM_EXPERIMENT_DIR: ${{ github.workspace }}/experiment
+    steps:
+      - uses: actions/checkout@v2
+# prerequisites for must
+      - run : sudo apt-get update && sudo apt-get install -y cmake git libxml2-dev openmpi-bin libopenmpi-dev
+      - run : sudo pip3 install numpy pandas lxml html5lib BeautifulSoup4
+# install MUST
```

Comment:
Should we remove these lines instead of commenting them out?

Reply:
?? they arent commented out it is yust a comment, what these lines do

Reply:
The placement of the commend seems weird. I mean those below. 20 - 25

Reply:
do we want the caching? as must runs more than 4h (need for oversubscribe) I think that it is not important if we spend a bit of time building it - Think we can remove it